setup.py

```diff
@@ -59,3 +137,5 @@
     # Use setuptools_scm to set the version number automatically from Git
     setup_requires=['setuptools_scm'],
 )
+
+shutil.rmtree('package_python')
```

Comment:
Check for existence of directory before removing it; remove it only if it exists, otherwise this will throw an error (perhaps not a likely scenario, but ya never know...)